imports/api/lists/methods.js

```diff
@@ -67,9 +67,9 @@ export const makePublic = new ValidatedMethod({
 export const updateName = new ValidatedMethod({
   name: 'lists.updateName',
   validate: new SimpleSchema({
```

Comment:
You could indeed pick it. The reason I used the schema method in some cases is entirely to allow giving the parameter a different name from that used in the schema as in `listId: Lists.simpleSchema().schema('_id'),`. 

In my own code I do a lot less of that than I did here, preferring to keep the schema's name for the parameter. i.e. I wouldn't call a 'text' parameter 'newText'. That allows me to use pick more often. 

The exception is _id. I find _id to be very undescriptive and I often have cases where a method needs to receive more than one _id such as a listId and a todoId. Because of that, I always use the schema method with _id instead of picking it.